linera-views/src/backends/dual.rs

```diff
@@ -7,15 +7,26 @@ use serde::{Deserialize, Serialize};
 use thiserror::Error;
 
 #[cfg(with_testing)]
-use crate::store::TestKeyValueStore;
+use crate::store::TestKeyValueDatabase;
 use crate::{
     batch::Batch,
     store::{
-        AdminKeyValueStore, KeyValueStoreError, ReadableKeyValueStore, WithError,
+        KeyValueDatabase, KeyValueStoreError, ReadableKeyValueStore, WithError,
         WritableKeyValueStore,
     },
 };
 
+/// A dual database.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct DualDatabase<D1, D2, A> {
+    /// The first database.
+    pub first_database: D1,
+    /// The second database.
+    pub second_database: D2,
+    /// Marker for the static root key assignment.
+    _marker: std::marker::PhantomData<A>,
+}
```

Comment:
So no rename `first`-> `shared` and `second`-> `exclusive`.

Reply:
In a next PR. It wouldn't be a rename